### PR TITLE
fix voters list bug

### PR DESCRIPTION
**Pull request by RoseRompkxm**

### CODE REVIEW

File: src/components/Vote/VoteOverview/VoteCard.js [hunk 1]
```diff
@@ -220,7 +228,11 @@ function VoteCard({
               </div>
             ))}
           </div>
-          {isViewAll && addressNumber > 3 && (
+          {/* view all is clicked and there are still empty lines to fill up */}
+          {!isViewAll && remainingToLoad > 0 && (
+            <Icon className="loading-icon" type="loading" />
+          )}
+          {isViewAll && addressNumber > 3 && remainingToLoad > 0 && (
```
Comment (kkirka) [inline src/components/Vote/VoteOverview/VoteCard.js]: Why 3 and not 4 though? I see it works correctly, just curious

File: src/components/Vote/VotingWallet.js [hunk 2]
```diff
@@ -165,10 +165,7 @@ function VotingWallet({
           <div className="flex align-center just-between">
             <div className="flex align-center">
               <img src={coinImg} alt="coin" />
-              <p className="content-value">
-                {getBefore(format(balance))}
-                <span>{getAfter(format(balance))}</span>
-              </p>
+              <p className="content-value">{format(balance)}</p>
```
Comment (kkirka) [inline src/components/Vote/VotingWallet.js:158]: Do we still need `getBefore` and `getAfter` functions to be defined then?

File: src/utilities/time.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import moment from 'moment';
+
+export const FORMAT_STRING = 'lll';
+
+export const formatTimeInSeconds = sec => {
+  return `${moment(sec * 1000).format(FORMAT_STRING)}`;
```
Comment (kkirka) [inline src/utilities/time.js:9]: Now we sometimes get Jan, 01, 1970. I propose we have sth like:

```suggestion
  if (!sec) {
    return '—';
  }
  return `${moment(sec * 1000).format(FORMAT_STRING)}`;
```

**Review by kkirka — APPROVED** [review]

File: src/components/Vote/VotingWallet.js (hunk 2 above)
Comment (RoseRompkxm) [reply src/components/Vote/VotingWallet.js:158]: no need for that, i will delete these useless functions

File: src/utilities/time.js (hunk 3 above)
Comment (RoseRompkxm) [reply src/utilities/time.js:9]: that's good suggestion, let me handle this

File: src/components/Vote/VoteOverview/VoteCard.js (hunk 1 above)
Comment (RoseRompkxm) [reply src/components/Vote/VoteOverview/VoteCard.js]: actually 'addressNumber > 3' is uselss now, i will delete it

**Pull request merged**